packages/libdbus/libdbus.spec

```diff
@@ -26,38 +26,51 @@ Requires: %{name}
 %autosetup -n dbus-%{version} -p1
 
 %build
-%cross_configure \
-  --disable-asserts \
-  --disable-doxygen-docs \
-  --disable-ducktype-docs \
-  --disable-tests \
-  --disable-xml-docs \
-  --disable-selinux \
-  --disable-systemd \
-  --with-xml=expat \
+CONFIGURE_OPTS=(
+ -Dasserts=false
+ -Dcontainers=false
+ -Dembedded_tests=false
+ -Dinstalled_tests=false
+ -Dmessage_bus=false
+ -Dstats=false
+ -Dtools=false
+ -Dtraditional_activation=false
+ -Duser_session=false
 
-%force_disable_rpath
+ -Dapparmor=disabled
+ -Ddoxygen_docs=disabled
+ -Dducktype_docs=disabled
+ -Dkqueue=disabled
+ -Dlaunchd=disabled
+ -Dlibaudit=disabled
+ -Dmodular_tests=disabled
+ -Dqt_help=disabled
+ -Drelocation=disabled
+ -Dselinux=disabled
+ -Dsystemd=disabled
+ -Dvalgrind=disabled
+ -Dx11_autolaunch=disabled
+ -Dxml_docs=disabled
 
-%make_build
+ -Dchecks=true
+)
+
+%cross_meson "${CONFIGURE_OPTS[@]}"
+%cross_meson_build
 
 %install
-%make_install
+%cross_meson_install
 
 rm -rf %{buildroot}%{_cross_docdir}/dbus/examples
 
 %files
 %license COPYING
 %{_cross_attribution_file}
 %{_cross_libdir}/*.so.*
-%exclude %{_cross_bindir}
-%exclude %{_cross_datadir}/dbus-1
```

Comment:
I'm curious to see the new rpm files list compared to the old, this looks like a nice cleanup!

Reply:
Here's the `rpm -qlp` diff. Only notable difference is the static library going away
```
❯ diff -u -p old new
--- old	2024-11-19 23:54:59.173307332 +0000
+++ new	2024-11-19 23:56:14.534202978 +0000
@@ -1,10 +1,10 @@
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3
-/aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3.37.0
+/aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3.38.1
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus/COPYING
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus/attribution.txt
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3
-/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3.37.0
+/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so.3.38.1
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus/COPYING
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/licenses/libdbus/attribution.txt
@@ -31,7 +31,6 @@
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include/dbus
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include/dbus/dbus-arch-deps.h
-/aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.a
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so
 /aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/pkgconfig/dbus-1.pc
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/include/dbus-1.0
@@ -57,6 +56,5 @@
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include/dbus
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/dbus-1.0/include/dbus/dbus-arch-deps.h
-/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.a
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/libdbus-1.so
 /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/pkgconfig/dbus-1.pc
```